client/src/components/CollegeShortlist.js

```diff
@@ -0,0 +1,103 @@
+import React, { Component } from 'react';
+import { Menu, Label, Card, Button, Icon } from 'semantic-ui-react';
+import CollegeShortlistModal from './CollegeShortlistModal';
+
+const STUDENT = "STUDENT"
+
+export default class CollegeShortlist extends Component {
+    constructor(props) {
+        super(props)
+        this.state = {
+            activeItem: "shortlist",
+            collegeModalOpen: false,
+            collegeShortlist: [],
+        }
+        this.openCollegeModal = this.openCollegeModal.bind(this);
+        this.closeCollegeModal = this.closeCollegeModal.bind(this);
+        var shortlist = this.shortlist.bind(this);
+    }
+
+    openCollegeModal() {
+        this.setState({
+            collegeModalOpen: true
+        })
+    }
+    closeCollegeModal() {
+        this.setState({
+            collegeModalOpen: false
+        }, () => {
+            this.props.refreshProfile(STUDENT, this.props.details._id)
+        })
+    }
+    shortlist(update) {
```

Comment:
This method could be called `addToShortlist` and it can update the `collegeShortlist` state variable here like ...